### PR TITLE
XIVY-12909 axios client codegen with orval

### PR DESCRIPTION
**Pull request by ivy-lmu**
- script for downloading openapi.yaml files from given engien url
- codegen axios client & adapt to new clients

### CODE REVIEW

File: orval.config.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'orval';
+
+export default defineConfig({
```
Comment (ivy-lmu) [inline orval.config.ts:3]: per default axios client is used. here we can also define client fetch for neo.

Comment (ivy-lli) [inline orval.config.ts:3]: maybe we could think about a filter, for only those api's we really use here? 
https://orval.dev/reference/configuration/input#tags

**Review by ivy-lli — APPROVED** [review]
Cool. Only thing which is missing I think is a build step where it is checked if the protocol is still compatible

**Comment by ivy-lmu** [comment]
> Cool. Only thing which is missing I think is a build step where it is checked if the protocol is still compatible

yes, will do it in a followup

File: orval.config.ts (hunk 1 above)
Comment (ivy-lmu) [reply orval.config.ts:3]: good one, I will check

File: orval.config.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'orval';
+
+const hooks = { afterAllFilesWrite: 'prettier --write' };
+const filters = { tags: ['web-ide'] };
```
Comment (ivy-lli) [inline orval.config.ts:4]: 👍

**Review by ivy-lli — APPROVED** [review]

**Pull request merged**